app/lib/theme/widgets/ContextualHelp.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
+import { faExclamationCircle } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Tooltip from "@mui/material/Tooltip";
+import { BC_GOV_LINKS_COLOR, TASKLIST_GREY_BG_COLOR } from "../colors";
+
+/**
+ * @param {string} title - The title of the tooltip to be displayed (can be HTML)
+ * @param {IconDefinition} icon - The icon to display in the tooltip; defaults to faExclamationCircle
+ * @param {string} placement - The placement of the tooltip; defaults to "top"
+ * @param {string} label - The label of the tooltip to be used as the aria-labelledby attribute; this gets passed by the parent component(FieldLabel)
+ * @returns {ReactElement} - The tooltip component
+ * To use this component, simply add "ui:tooltip" object to the uiSchema of the field you want to add a tooltip to.
+ * Example:
+ * "ui:tooltip": {
+ *   "title": "<div>Some content here with <a>Links</a> and other HTML tags</div>",
+ *   "icon": faExclamationCircle,
+ *   "placement": "top"
+ * }
+ */
+
+interface ContextualHelpProps {
+  title: string;
+  icon?: IconDefinition;
+  placement?:
+    | "top"
+    | "bottom"
+    | "left"
+    | "right"
+    | "bottom-end"
+    | "bottom-start"
+    | "left-end"
+    | "left-start"
+    | "right-end"
+    | "right-start"
+    | "top-end"
+    | "top-start";
+  label: string;
+}
+
+const ContextualHelp: React.FC<ContextualHelpProps> = ({
+  title,
+  icon = faExclamationCircle,
+  placement = "bottom",
+  label,
+}) => {
+  // generate a unique id for the tooltip based on the label
+  const tooltipId = `${label.replace(/\s+/g, "-").toLowerCase()}-tooltip`;
+  return (
+    <>
+      <Tooltip
+        title={<span dangerouslySetInnerHTML={{ __html: title }} />}
```

Comment:
I tried it but didn't work since we want to pass HTML code as the text. Also, I think it's safe to use `dangerouslySetInnerHTML` here since we are not getting user input here and simply passing the text from the UI schema to the client side.

Reply:
I read that thing about it being safe if you never get user input, but I think I'd still like a second opinion on this one since it's a potential security thing. @mikevespi or @dleard, could you please weigh in?

Reply:
I would love to change this if there's a workaround to this. I've found a way to parse HTML as text by adding the below package to the code but I'm not sure if it's really necessary:
https://www.npmjs.com/package/html-react-parser